src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+const Footer = () => {
+  return (
+    <footer>
+      <p>
+        Created at{" "}
```

Comment:
you should put your name as well and link it to your github or portfolio! If this app was mine, I would be very proud and won't forget to my name on it! 😄

Reply:
Good point! Will do 🙂